src/assets/styles/components/_audio-tactile-subsection.scss

```diff
@@ -55,9 +57,9 @@
         min-width: 3.75rem;
 
         svg {
-            --icon-arrowColour: var(--fl-buttonFgColor, #{$sectionBorderColour});
+            --icon-arrowColour: var(--fl-buttonFgColor, #{var(--darkRed)});

```

Comment:
`#{var(--darkRed)}` is not necessary, one can use `var(--darkRead)` without the `#{}`. Using the hash curly brace syntax is Sass string templating, which was needed when a Sass variable was used, but is no longer needed now that a CSS custom property is used.